js/iframe-coordinator/src/messages/validationUtils.ts

```diff
@@ -0,0 +1,19 @@
+import { Guard } from 'decoders';
+
+export function createMessageValidator<T>(
```

Comment:
Moved a bunch of boilerplate logic here that sprung up once i moved the validates into their own message files.